src/Devices.js

```diff
@@ -5,7 +5,7 @@ import pcsclite from 'pcsclite';
 import Device from './Device';
 
 
-const pcsc = pcsclite();
+// const pcsc = pcsclite();
```

Comment:
if this line is actually not required then let's delete it rather than comment it out :-)